server/http_handler.go

```diff
@@ -284,12 +302,20 @@ func (t *tikvHandlerTool) schema() (infoschema.InfoSchema, error) {
 	return domain.GetDomain(session.(sessionctx.Context)).InfoSchema(), nil
 }
 
-func (t *tikvHandlerTool) handleMvccGetByHex(params map[string]string) (interface{}, error) {
+func (t *tikvHandlerTool) handleMvccGetByHex(params map[string]string) (*mvccKV, error) {
 	encodedKey, err := hex.DecodeString(params[pHexKey])
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
-	return t.GetMvccByEncodedKey(encodedKey)
+	data, err := t.GetMvccByEncodedKey(encodedKey)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	keyLocation, err := t.RegionCache.LocateKey(tikv.NewBackoffer(context.Background(), 500), encodedKey)
```

Comment:
There are many similar code like this. It is better to extract a function for it.

Reply:
PTAL @jackysp